### PR TITLE
DEPL-10919 added help files for the pipeline steps

### PR DESCRIPTION
**Pull request by mehrat**

### CODE REVIEW

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-environmentId.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div>
+    Id of the environment in xl-deploy on which artifact needs to be deployed. Jenkin's environment variables can be used to specify this value.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-environmentId.html]: ID of the environment in XL Deploy to which the artifact should be deployed. Jenkin's environment variables can be used to specify this value.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-packageid.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div>
+    Id of the package in xl-deploy which needs to be deployed. Jenkin's environment variables can be used to specify this value.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-packageid.html]: ID of the package in XL Deploy that should be deployed. Jenkins environment variables can be used to specify this value.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-servercredentials.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div>
+    xl-deploy server credentials used to connect xl-deploy server.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-servercredentials.html]: XL Deploy server credentials used to connect to the XL Deploy server.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-servercredentials.html [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+    This needs to be configured under Managed Jenkins >> Configure System >> XL Deploy.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help-servercredentials.html]: This must be configured under Manage Jenkins >> Configure System >> XL Deploy.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help.html [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<div>
+    Adds a step to deploy artifacts on environments in xl-deploy server using jenkins.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployDeployStep/help.html]: Adds a step to deploy artifacts on environments in the XL Deploy server using Jenkins.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-artifactsPath.html [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<div>
+    Path to artifact directory relative to workspace. Jenkin's environment variables can be used to specify this value.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-artifactsPath.html]: Path to the artifact directory, relative to the workspace. Jenkins environment variables can be used to specify this value.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-darPath.html [hunk 7]
```diff
@@ -0,0 +1,4 @@
+<div>
+    Path for newly created dar file relative to workspace. Jenkin's environment variables can be used to specify this value.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-darPath.html]: Path for the newly created DAR file, relative to the workspace. Jenkins environment variables can be used to specify this value.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-manifestPath.html [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<div>
+    Path of deployit-manifest.xml relative to workspace. Jenkin's environment variables can be used to specify values inside manifest file.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help-manifestPath.html]: Path of the deployit-manifest.xml file, relative to the workspace. Jenkin's environment variables can be used to specify values in the manifest file.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help.html [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<div>
+    Adds a step to package a dar file.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPackageStep/help.html]: Adds a step to package a DAR file.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-darPath.html [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<div>
+    Path to dar file(relative to workspace) which needs to be published on xl-deploy server.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-darPath.html]: Path to the DAR file (relative to the workspace) that should be published on the XL Deploy server.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-serverCredentials.html [hunk 11]
```diff
@@ -0,0 +1,4 @@
+<div>
+    xl-deploy server credentials used to connect xl-deploy server.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-serverCredentials.html]: XL Deploy server credentials used to connect to the XL Deploy server.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-serverCredentials.html [hunk 12]
```diff
@@ -0,0 +1,4 @@
...
+    This needs to be configured under Managed Jenkins >> Configure System >> XL Deploy.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help-serverCredentials.html]: This must be configured under Manage Jenkins >> Configure System >> XL Deploy.

File: src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help.html [hunk 13]
```diff
@@ -0,0 +1,3 @@
+<div>
+    Adds a step to publish dar file to xl-deploy server using jenkins.
```
Comment (amyjohnston) [inline src/main/resources/com/xebialabs/deployit/ci/workflow/XLDeployPublishStep/help.html]: Adds a step to publish the DAR file to the XL Deploy server using Jenkins.

**Comment by mehrat** [comment]
Added all the changes suggested by Amy

**Pull request merged**